pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -141,12 +141,14 @@ func coreBootstrap(initConfiguration *kubeadmapi.InitConfiguration, bootstrapCon
 	err = target.Apply(
 		bootstrapConfiguration,
 		"kubeadm.reset",
+		"kubelet.rootca.create",
```

Comment:
As mentioned earlier, I think we can make this a regular function call, no need for target there.